### PR TITLE
[SPARK-40463][INFRA] Update gpg's keyserver

### PR DESCRIPTION
**Pull request by wangyum**
### What changes were proposed in this pull request?

Update gpg's keyserver to hkps://keyserver.ubuntu.com.

### Why are the changes needed?

`keyserver.ubuntu.com` will throw an exception:
```bash
yumwang@LM-SHC-16508156 create-release % gpg --keyserver keyserver.ubuntu.com --recv-key E298A3A825C0D65DFD57CBB651716619E084DAB9
gpg: keyserver receive failed: End of file
yumwang@LM-SHC-16508156 create-release % gpg --keyserver hkp://keyserver.ubuntu.com:80 --recv-key E298A3A825C0D65DFD57CBB651716619E084DAB9
gpg: key 51716619E084DAB9: public key "Michael Rutter <marutter@gmail.com>" imported
gpg: Total number processed: 1
gpg:               imported: 1
```

### Does this PR introduce _any_ user-facing change?

No.

### How was this patch tested?

Manually test.

### CODE REVIEW

File: dev/create-release/spark-rm/Dockerfile
Comment (Yikun) [inline dev/create-release/spark-rm/Dockerfile:56]: Another choice might be:

```suggestion
  gpg --keyserver hkps://keyserver.ubuntu.com --recv-key E298A3A825C0D65DFD57CBB651716619E084DAB9 && \
```

```
$ gpg --keyserver hkps://keyserver.ubuntu.com --recv-keys E298A3A825C0D65DFD57CBB651716619E084DAB9
gpg: key 51716619E084DAB9: public key "Michael Rutter <marutter@gmail.com>" imported
gpg: Total number processed: 1
gpg:               imported: 1
```

I also hitted this error before

[1] https://github.com/search?l=Dockerfile&q=org%3Adocker-library+keys.openpgp.org&type=Code

**Review by HyukjinKwon — APPROVED** [review]
LGTM if it works. cc @xinrong-meng FYI who faced this problem before.

**Review by Yikun — APPROVED** [review]
Both OK and LGTM

File: dev/create-release/spark-rm/Dockerfile
Comment (dongjoon-hyun) [inline dev/create-release/spark-rm/Dockerfile:56]: We have the same code in another place.
https://github.com/apache/spark/blob/32fbd7e83252f96df9c78f2f15f3917167821a12/dev/infra/Dockerfile#L52

Comment (dongjoon-hyun) [reply dev/create-release/spark-rm/Dockerfile:56]: Both images are based on `Ubuntu 20.04` (Focal Fossa). If we need to change it, we had better consistent.

**Review by dongjoon-hyun — COMMENTED** [review]
Never mind. I deleted this outdated comment in this place.

**Review by dongjoon-hyun — APPROVED** [review]
+1, LGTM (Pending CIs)

**Comment by dongjoon-hyun** [comment]
Oh, @wangyum . It seems that you made an accidental commit on the `master` branch.
- https://github.com/apache/spark/commit/694cac63da3bfa651132eca9fee3278544616dc3

**Comment by wangyum** [comment]
> Oh, @wangyum . It seems that you made an accidental commit on the `master` branch.
> 
> * [694cac6](https://github.com/apache/spark/commit/694cac63da3bfa651132eca9fee3278544616dc3)

Sorry. I didn't which to my branch.

**Comment by dongjoon-hyun** [comment]
Ya, the accident happens sometime. No worry. If CI succeeds, nobody gets hurt.

**Comment by wangyum** [comment]
Could we force push to overwrite that commit?

**Comment by dongjoon-hyun** [comment]
No force-push, @wangyum . We already have another commit on top of yours.

**Comment by wangyum** [comment]
OK

**Comment by HyukjinKwon** [comment]
Let me just revert and merge this PR in (just for the sake of trackability).

**Comment by dongjoon-hyun** [comment]
Yep, reverting is possible. I'll leave it to you, @HyukjinKwon .

**Comment by HyukjinKwon** [comment]
Merged to master.

**Pull request closed without merging**
